manifests/password.pp

```diff
@@ -40,18 +40,18 @@
   }
 
   file { "$password_config_file":
-    content => $password_content,
     ensure  => present,
+    content => $password_content,
     require => Package[$package_name],
     notify  => Service[$virtual_service],
     tag     => "splunk_password",
   }
 
   file { "$secret_file":
+    ensure  => present,
```

Comment:
Somehow, I thought this was a lint error in itself and we preferred `ensure => file`?